Sources/TuistDependencies/SwiftPackageManager/Models/WorkspaceState.swift

```diff
@@ -0,0 +1,28 @@
+/// The Swift Package Manager workspace information.
+/// It decodes data encoded from WorkspaceState.swift: https://github.com/apple/swift-package-manager/blob/ce50cb0de101c2d9a5742aaf70efc7c21e8f249b/Sources/Workspace/WorkspaceState.swift
+/// In particular, we are interested in the ManagedDependency.swift: https://github.com/apple/swift-package-manager/blob/ce50cb0de101c2d9a5742aaf70efc7c21e8f249b/Sources/Workspace/ManagedDependency.swift
+/// Fields not needed by tuist are commented out and not decoded at all.
+struct WorkspaceState: Decodable, Equatable {
+    /// The products declared in the manifest.
+    let object: Object
+}
+
+extension WorkspaceState {
```

Comment:
Considering you are using extensions to organize the code in blocks, what about making things easier for the compiler and keeping everything in the declaration of the struct?

```swift
struct WorkspaceState: Decodable, Equatable {
  // All the code here
}
```